privacyidea/models.py

```diff
@@ -3267,3 +3267,12 @@ def save(self):
         else:
             ret = self.id
         return ret
+
+
+class NodeName(db.Model, TimestampMethodsMixin):
+    __tablename__ = "nodename"
+    # TODO: we can use the UUID type here when switching to SQLAlchemy 2.0
```

Comment:
Just think about my other comment about TODOs. Thanks.

Reply:
I do think that manually converting the UUID from/to strings is a hacky workaround and should be revised when switching to SQLAlchemy 2.0